ui/org.eclipse.pde.core/schema/dynamicSource.exsd

```diff
@@ -0,0 +1,125 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<!-- Schema file written by PDE -->
+<schema targetNamespace="org.eclipse.pde.core" xmlns="http://www.w3.org/2001/XMLSchema">
+<annotation>
+      <appInfo>
+         <meta.schema plugin="org.eclipse.pde.core" id="dynamicSource" name="Dynamic Source Locations"/>
+      </appInfo>
+      <documentation>
+         This extension point allows PDE to find source archives for libraries at runtime.
+      </documentation>
+   </annotation>
+
+   <element name="extension">
+      <annotation>
+         <appInfo>
+            <meta.element />
+         </appInfo>
+      </annotation>
+      <complexType>
+         <choice minOccurs="1" maxOccurs="unbounded">
+            <element ref="locator"/>
+         </choice>
+         <attribute name="point" type="string" use="required">
+            <annotation>
+               <documentation>
+                  
+               </documentation>
+            </annotation>
+         </attribute>
+         <attribute name="id" type="string">
+            <annotation>
+               <documentation>
+                  
+               </documentation>
+            </annotation>
+         </attribute>
+         <attribute name="name" type="string">
+            <annotation>
+               <documentation>
+                  
+               </documentation>
+               <appInfo>
+                  <meta.attribute translatable="true"/>
+               </appInfo>
+            </annotation>
+         </attribute>
```

Comment:
```suggestion
         <attribute name="point" type="string" use="required"/>
```
The Schema-Editor adds the `name` and `id` attributes by default. But they seem to be unused and in a quick check it also worked without.
So can't it simply be removed?

Reply:
@laeubi do you know if my assumption is right?
I wonder if the schema is even evaluated at runtime? Because when I delete it everything still works. It is just the PDE Extension Editor that complains and does not know how to handle the attributes/childs.

Reply:
I have no clue maybe @merks know about ti, I'm from "the other side" that is using OSGi service for extends my code, I only use extension points with eclipse and even there try to avoid it :-)